packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -423,6 +425,17 @@ class Tabs extends Foundation<TabsHandledProps, TabsUnhandledProps, TabsState> {
               );
     }
 
+    /**
+     * Need to filter out none truthy results for Preact.
+     * Can remove if below gets merged in.
+     * https://github.com/preactjs/preact-compat/pull/461
+     */
+    private filterChildren(nodes: any): React.ReactNode {
```

Comment:
Can you explain why you're only doing this filter if there are props or items? Don't you only want to filter when `nodes` is an array?

Reply:
yup your right, updated